docs/DeveloperGuide.md

```diff
@@ -727,6 +727,14 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Non-Functional Requirements
 
+1. Should work on any mainstream OS as long as it has Java 11 or above installed.
+
+1. Should be able to recover latest state of CHIM (before the command that causes the crash was executed) if the application crashes.
```

Comment:
numbering is all 1?

Reply:
If you view in markdown, the numbering will be autonumbered correctly